bug: temporary fix for deadlink in Jenkins on Google Cloud tutorial (#7602)
jenkins-infra/jenkins.io: content/doc/tutorials/tutorials-for-installing-jenkins-on-Google-Cloud.adoc

* **Packer** tool for creating images.

Dive into the tutorial for installing jenkins on Google Cloud here:

Watch the video tutorial here:
video::Zy_FQEYkaRw[youtube, width=640, height=360]

> 

Reviewer: ```suggestion
```

Author: removed line 22

> For a reference to the video, check out this https://cloud.google.com/blog/products/gcp/using-jenkins-on-google-compute-engine-for-distributed-builds[user guide], which provides the approach required.

Reviewer: ```suggestion
For a reference to the video, check out this https://cloud.google.com/blog/products/gcp/using-jenkins-on-google-compute-engine-for-distributed-builds[blog post], which provides the approach required.
```

Reviewer: One more change needed

Author: done how does it look now

* **Packer** tool for creating images.

> Dive into the tutorial for installing jenkins on Google Cloud here:

Reviewer: ```suggestion
### Dive into the tutorial for installing jenkins on Google Cloud
```

> Watch the video tutorial here:

Reviewer: ```suggestion
* Watch the video tutorial here:
```

> video::Zy_FQEYkaRw[youtube, width=640, height=360]

Reviewer: ```suggestion

video::Zy_FQEYkaRw[youtube, width=640, height=360,  align="center"]

```
